egs/librispeech_100/asr1/conf/tuning/train_conformer_transducer.yaml

```
@@ -0,0 +1,55 @@
+# minibatch related
+batch-size: 16
+maxlen-in: 512
+maxlen-out: 150
+
+# optimization related
+criterion: loss
+early-stop-criterion: "validation/main/loss"
+sortagrad: 0
+opt: noam
+transformer-lr: 5.0
+transformer-warmup-steps: 25000
+epochs: 100
+patience: 0
+accum-grad: 8
+grad-clip: 5.0
+
+# network architecture
+## general
+custom-enc-positional-encoding-type: rel_pos
+custom-enc-self-attn-type: rel_self_attn
+custom-enc-pw-activation-type: swish
+## encoder related
+etype: custom
+custom-enc-input-layer: vgg2l
```

```suggestion
custom-enc-input-layer: vgg2l
custom-enc-input-dropout-rate: 0.3
```

It's easily missed but we introduced `custom-enc-input-dropout-rate` and `custom-enc-input-pos-enc-dropout-rate` in `v.0.10.4`. Previously, the value was set automatically according to the encoder architecture parameters. The most dominant value for `dropout-rate` (or `pos-dropout-rate)` was assigned to the input (pos. enc.) dropout layer.

Also, note that we found some small performance degradation in Aishell-1 between `v.0.10.3` and `v.0.10.5`. Not sure if it's related to changes for transducer though.
We're investigating so feel free to notify me if you have some performance issue or a comparison between these versions (IIRC your initial experiments with transducer started from `v.0.10.3`)!

---

Thanks for letting me know about the dropout configuration! I'll be careful from now on.

I haven't run experiments for Transducer yet, so I'll let you know the results as soon as I get them!